api/src/main/java/io/opencensus/trace/Span.java

```diff
@@ -52,7 +51,7 @@
   public enum Options {
     /**
      * This option is set if the Span is part of a sampled distributed trace OR the {@link
-     * StartSpanOptions#getRecordEvents()} is true.
+     * SpanBuilder#setRecordEvents(boolean)} is true.
```

Comment:
Should this be "`OR {@link SpanBuilder#setRecordEvents(boolean)} was called with true.`"?